vllm/hpu/xops.py

```diff
@@ -0,0 +1,103 @@
+###############################################################################
+# Copyright (C) 2023 Habana Labs, Ltd. an Intel Company
+# All Rights Reserved.
+#
+# Unauthorized copying of this file or any element(s) within it, via any medium
+# is strictly prohibited.
+# This file contains Habana Labs, Ltd. proprietary and confidential information
+# and is subject to the confidentiality and license agreements under which it
+# was provided.
+#
+###############################################################################
```

Comment:
I'm not sure how relevant is that copyright header in open-source code

Reply:
so leave only "# Copyright (C) 2024 Habana Labs, Ltd. an Intel Company"?

Reply:
yes